pkgs/desktops/cinnamon/cjs/default.nix

```diff
@@ -0,0 +1,67 @@
+{ autoconf-archive
+, autoreconfHook
+, dbus-glib
+, fetchFromGitHub
+, gobject-introspection
+, pkgconfig
+, spidermonkey_52
+, stdenv
+, wrapGAppsHook
+, python3
+, cairo
+, gnome3
+, xapps
+, keybinder3
+, upower
+}:
+
+stdenv.mkDerivation rec {
+  pname = "cjs";
+  version = "4.4.0";
+  enableParallelBuilding = true;
+
+  src = fetchFromGitHub {
+    owner = "linuxmint";
+    repo = pname;
+    rev = version;
+    sha256 = "0q5h2pbwysc6hwq5js3lwi6zn7i5qjjy070ynfhfn3z69lw5iz2d";
+  };
+
+  buildInputs = [
```

Comment:
are we sure these are all correct?
https://github.com/linuxmint/cjs/blob/master/debian/control#L9

`dbus-glib` I don't see.